Update concepts-limits.md

arwinlashawn opened the pull request:
Found out today that it is actually possible to have the database + container count limit per account increased, and so I think this documentation should be updated accordingly to reflect that. Please reach out to me directly via alias arwinlashawn if you need more details on this.

arwinlashawn committed:
Update concepts-limits.md

Found out today that it is actually possible to have the serverless container count limit increased, and so I think this documentation should be updated accordingly to reflect that. Please reach out to me directly via alias arwinlashawn if you need more details on this.

seesharprun commented on articles/cosmos-db/concepts-limits.md:
```diff
@@ -152,9 +152,11 @@ Here's a list of limits per account.
+| Maximum number of databases and containers per account  | 100 ¹ |
 | Maximum number of regions | 1 (Any Azure region) |
 
+¹ You can increase this limit by creating an [Azure Support request](create-support-request-quota-increase.md) up to 500 max.
```
```suggestion
¹ To increase this limit, create an [Azure support request](create-support-request-quota-increase.md).
```

Minor edits. To increase the longevity of this article, it's best to not publish the support max as this number may increase in the future.

arwinlashawn replied:
@seesharprun  Agreed! Have removed the limit, but maintained the wording as per the rest of the document - keeping the footer note consistent e.g. https://learn.microsoft.com/en-us/azure/cosmos-db/concepts-limits#:~:text=%C2%B9%20You%20can%20increase%20Maximum%20RUs%20per%20container%20or%20database%20by%20filing%20an%20Azure%20support%20ticket.

arwinlashawn committed:
Update concepts-limits.md

Incorporating suggestions but keeping the same structure as per the rest of the document: https://learn.microsoft.com/en-us/azure/cosmos-db/concepts-limits#:~:text=%C2%B9%20You%20can%20increase%20Maximum%20RUs%20per%20container%20or%20database%20by%20filing%20an%20Azure%20support%20ticket.

Court72 commented on articles/cosmos-db/concepts-limits.md:
```diff
@@ -152,9 +152,11 @@ Here's a list of limits per account.
+| Maximum number of databases and containers per account  | 100 ¹ |
 | Maximum number of regions | 1 (Any Azure region) |
 
+¹ You can increase this limit by creating an [Azure Support request](create-support-request-quota-increase.md).
```
```suggestion
¹ You can increase this limit by creating an [Azure Support request](nosql/create-support-request-quota-increase.md).
```

Court72 committed:
Update articles/cosmos-db/concepts-limits.md
